loop_rate:        10.0 # The frequency in Hz at which to run the control loop.
queue_size:       1    # The size of the message queue used for publishing and subscribing to topics.
precision:        2    # Number of decimals to create when converting to meters.
origin_init_drop: 10   # Number of messages to drop initially before setting the origin.
